### ISSUE TITLE
[TF2] STEAM validation rejected

### ISSUE THREAD

**Issue by BitmapDummy** [ValveSoftware/Source-1-Games#5627]
When attempting to join a community server, this error message appears. This was after the gameserver was updated, new login tokens, new account identity credentials. What am I missing?

**Comment by iBoonie** [comment]
This is caused by a sourcemod extension, specifically plugins using: https://forums.alliedmods.net/showthread.php?t=162489

**Comment by BitmapDummy** [comment]
> This is caused by a sourcemod extension, specifically plugins using: https://forums.alliedmods.net/showthread.php?t=162489

So do we just disable plugins?

**Comment by BitmapDummy** [comment]
And this has nothing to do with login tokens or those `cl_gameserver_create_identity` things, right?

**Comment by Nik1Toxic** [comment]
Same issue. Disabling Connect extension helped

**Comment by BitmapDummy** [comment]
How do you do that?

-------- Original Message --------

**Comment by BitmapDummy** [comment]
Yes, that worked. STAC seems to break without it, but LilAC seems fine.

**Comment by Mooshua** [comment]
This issue is not caused by connect! This is a bug in the Steam authentication flow which allows an unauthenticated client to play for extended amounts of time, likely due to a transient failure in the Steam client.

I have no reproduction but rebooting the Steam service always fixes it. Going offline/online also worked the one time I tried it.

**Disabling connect is not a true fix,** the bug will still occur, players will not be authenticated and they will eventually be booted by VAC if the client doesn’t sort itself out in time.
